flexget/plugins/sites/lostfilm.py

```diff
@@ -47,147 +37,128 @@ class LostFilm(object):
 
       lostfilm:
         url: <url>
+        pages: <number>
     """
 
     schema = {
         'type': ['boolean', 'object'],
         'properties': {
             'url': {'type': 'string', 'format': 'url'},
+            'pages': {'type': 'integer'}
         },
         'additionalProperties': False
     }
 
     def build_config(self, config):
-        """Set default url to config"""
+        """Set default url and pages config"""
         if isinstance(config, bool) and config is True:
```

Comment:
`config is True` is unnecessary

Reply:
Here I've thinked of a case when the plugin is disabled with `config` eq to `false` (i.e. temporarily). What is the right way to do this?

Reply:
See the comment below